taskcluster/host-build.sh

```diff
@@ -21,6 +21,7 @@ do_deepspeech_binary_build
 
 do_deepspeech_python_build
 
+do_deepspeech_decoder_build
```

Comment:
Why do we need this one ? In the past, there was a `decoder-build.sh` call to build the `libctc_decoder_with_kenlm.so` for packaging it, but this one does not get packaged, right?

Reply:
Good point, I don't think this is needed.